test/test_quantized_tensor.py

```diff
@@ -124,9 +124,8 @@ def test_qtensor_creation(self):
         self.assertEqual(q.q_zero_point(), q_el.q_zero_point())
         self.assertEqual(q.dtype, q_el.dtype)
 
-        # create via empty_like but change the dtype (currently not supported)
-        with self.assertRaises(RuntimeError):
-            torch.empty_like(q, dtype=torch.qint8)
+        # create via empty_like but change the dtype
+        self.assertEqual(torch.qint8, torch.empty_like(q, dtype=torch.qint8).dtype)
```

Comment:
this definitely shouldn't work -- empty doesn't work with `torch.qint8` so empty_like shouldn't.